### ISSUE TITLE
GeForce now issue

### ISSUE THREAD

**Issue by qwaser2** [mdnpascual/ToF-Fishing-Bot#17]
would be great if it can work with GeForce now,the Executable for GeForce now  are "GeForceNOW.exe",i also attched link to images from task manager
![geforce now 1](https://user-images.githubusercontent.com/33985348/233767892-900f52fa-bacc-4cc9-ab15-5d3165f538af.PNG)
![geforce now 2](https://user-images.githubusercontent.com/33985348/233767895-e5163075-0696-4f56-bc2d-c329be3dcacf.PNG)
ty

**Comment by mdnpascual** [comment]
Ah shit, it's multiple .exe so the new v2 version won't work with geforceNow.

What I would suggest is to use the old version https://github.com/mdnpascual/ToF-Fishing-Bot/releases/download/1.03/ToF_Fishing_Bot_v1_0_2_experimental.zip which unfortunately doesn't have the QOL updates.

tutorial to setup old version: https://www.youtube.com/watch?v=cnyKoxRe9cA

**Comment by qwaser2** [comment]
ty i ll give it try

**Comment by zere34** [comment]
I have same problem to GeForce now but the bar are  not moving left and right to old version and upper left and lower right are reset after restart exe. It's possible to version not requiere .exe but just on windows on fullscreen ? A friend need too.  :)

**Comment by qwaser2** [comment]
> I have same problem to GeForce now but the bar are not moving left and right to old version and upper left and lower right are reset after restart exe. It's possible to version not requiere .exe but just on windows on fullscreen ? A friend need too. :)

ye i tried aswell it didnt work ,i dont know anything about coding but if there was way for new version  to not check game status would be great

**Comment by mdnpascual** [comment]
The old version (not version 2) doesn't check .exe and sends the keyboard and mouse output via the window that is currently on focus as long as it is ran as admin. 

If that still doesn't work then the GeForce now app is doing something else that is not accepting the keypresses and mouse presses of the old version

**Comment by zere34** [comment]
Yes I understand, yes Geforce now can receive key press information, I use the rewasd software which allows you to bind and make macros on controller for the game
